evennia/web/webclient/static/webclient/js/plugins/hotbuttons.js

```diff
@@ -0,0 +1,141 @@
+/*
+ *
+ * Assignable 'hot-buttons' Plugin
+ *
+ * This adds a bar of 9 buttons that can be shift-click assigned whatever is in the textinput buffer, so you can simply
+ * click the button again and have it execute those commands, instead of having to type it all out again and again.
+ *
```

Comment:
Since this is in effect an inactive "contrib", I think this intro needs a mention of 

- The fact that this is disabled by default
- How to practically activate it in the webclient